src/uct/ib/base/ib_md.h

```diff
@@ -357,13 +379,34 @@ typedef struct uct_ib_md_ops_entry {
     int                         priority;
 } uct_ib_md_ops_entry_t;
 
+#define uct_ib_md_ops_register(_name) \
+    uct_ib_md_ops_##_name##_ctor();
+
+#define UCT_IB_MD_OPS_REG_DECL(_name) \
+    void uct_ib_md_ops_##_name##_ctor(void);
+
+#define UCT_IB_MD_OPS_INIT(_md_ops, _priority, _name) \
```

Comment:
can we remove UCT_IB_MD_OPS macro?
seems like duplication - it also defines uct_ib_md_ops_entry_t

Reply:
> can we remove UCT_IB_MD_OPS macro?
> seems like duplication - it also defines uct_ib_md_ops_entry_t

UCT_IB_MD_OPS is replaced by new macro